CHANGELOG.md

```diff
@@ -9,7 +9,121 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 Note: Changes are now automatically tracked in [GitHub](https://github.com/sqlfluff/sqlfluff/releases) and will be copied in here on each release (please remember to update the issues and contributors to links!). There is no need to manually edit this file going forward.
 -->
 
-## [0.9.1] - 2022-01-08]
+
+## [0.9.2] - 2022-01-24
+
+## Highlights
+
+We are please to include over 80 fixes in this release, and welcome seven new contributors.
```

Comment:
Shouldn't this be "We are pleased to include over 80 fixes in this release and welcome seven new contributors"?  Sorry to nitpick -- love everything you do for the library @tunetheweb!

Reply:
Thanks! - fixed.